src/Text/StringTemplates/Templates.hs

```diff
@@ -170,3 +179,11 @@ instance (Applicative m, Monad m) => TemplatesMonad (TemplatesT m) where
   getTextTemplatesByLanguage lang = TemplatesT $ do
     (_, ts) <- ask
     return $ localizedVersion lang ts
+
+instance MonadError e m => MonadError e (TemplatesT m) where
+  throwError = lift . throwError
+  catchError = liftCatch catchError
+    where
+      liftCatch :: Catch e m a -> Catch e (TemplatesT m) a
+      liftCatch f m h = TemplatesT . ReaderT $ \r ->
+        f (runReaderT (unTT m) r) (\e -> runReaderT (unTT $ h e) r)
```

Comment:
You don't need to strip the ReaderT here

Reply:
Oh yeah, of course, cool :)